### PR TITLE
Changed babel options to avoid extra lines in report (closes #1226)

### LINKED ISSUE #1226: Incorrect callstack for test error
### Are you requesting a feature or reporting a bug?

bug

### What is the current behavior?

```sh
         at <anonymous> (d:\temp\help.js:2:11)
!!!!     at step (d:\temp\help.js:1:538)
!!!!     at <anonymous> (d:\temp\help.js:1:778)
!!!!     at <anonymous> (d:\temp\help.js:1:439)
!!!!     at exports.default (d:\temp\help.js:4:42)
         at <anonymous> (d:\temp\test1.js:9:11)
!!!!     at step (d:\temp\test1.js:3:364)
```

### What is the expected behavior?

```sh
         at <anonymous> (d:\temp\help.js:2:11)
         at <anonymous> (d:\temp\test1.js:9:11)
```

### How would you reproduce the current behavior (if this is a bug)?

run the test

#### Provide the test code and the tested page URL (if applicable)

Tested page URL: any

Test code

```js
// test.js
import help from './help.js';

fixture `Fixture`
    .page `http://devexpress.github.io/testcafe/example/`;

test('test', async t => {
    await help(t);
})

// help.js
export default async function (t) {
    throw new Error('error');
}
```

### Specify your

* operating system: win7
* testcafe version: 0.13.0-alpha2
* node.js version: `7.4.0`, `6.9.1`, `0.10.47`

### PR DESCRIPTION
**Pull request by georgiy-abbasov**
\cc @inikulin, @AlexanderMoskovkin

### CODE REVIEW

**Comment by georgiy-abbasov** [comment]
It won't work, [re-export still broken](https://github.com/babel/babel/issues/2877).

**Comment by georgiy-abbasov** [comment]
Asked for help on Babel forum. Anyway, i'm still working on it.
https://discuss.babeljs.io/t/passperpreset-option-brokes-transformruntime/919

**Comment by inikulin** [comment]
Seems like approach from https://github.com/DevExpress/testcafe/pull/1236/commits/8603d492c2fb62d9a30ba68ce37fef043e577d0d worked, why not stick with it?

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
@inikulin, i've got some problems with promises in server tests on v0.10

**Comment by inikulin** [comment]
@georgiy-abbasov We can enable polyfills for node 10 it's not a big deal: we'll have problem with callsites on node 10, but it's already in maintenance mode and we're about to drop support for it.

**Comment by georgiy-abbasov** [comment]
@inikulin, ok, we can do it, but it seems, that last approach works too.

**Comment by inikulin** [comment]
@georgiy-abbasov hmm, didn't see the last one. Looks good if it works

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
FPR

File: test/server/compiler-test.js [hunk 1]
```diff
@@ -1341,5 +1342,47 @@ describe('Compiler', function () {
...
+                .getTests()
+                .then(function (tests) {
+                    var test = tests[0];
+
+                    return test.fn(testRunMock);
+                });
+        });
+
+        it('Incorrect callsite stack in error report if "import" is used (GH-1226)', function () {
+            this.timeout(5000);
```
Comment (AlexanderMoskovkin) [inline test/server/compiler-test.js]: we can remove the timeout here because it's defined at the top-level

**Pull request merged**